### PR TITLE
Tupaia mobile quick wins

### PR DESCRIPTION
**Pull request by tcaiger**
### Issues:

- [3088](https://app.zenhub.com/workspaces/active-sprints-5eea9d3de8519e0019186490/issues/beyondessential/tupaia-backlog/3088) Fix Search
- [3171](https://app.zenhub.com/workspaces/active-sprints-5eea9d3de8519e0019186490/issues/beyondessential/tupaia-backlog/3171) Hide mini-map
- [3170](https://app.zenhub.com/workspaces/active-sprints-5eea9d3de8519e0019186490/issues/beyondessential/tupaia-backlog/3170) Open dashboard by default
- [3086](https://app.zenhub.com/workspaces/active-sprints-5eea9d3de8519e0019186490/issues/beyondessential/tupaia-backlog/3086) Improve project selection
---

### Screenshots:
See issue tickets

### CODE REVIEW

File: packages/web-frontend/src/actions.js [hunk 1]
```diff
@@ -904,13 +903,6 @@ export function toggleDashboardSelectExpand() {
   };
 }
 
-export function setMobileDashboardExpanded(shouldExpand) {
```
Comment (tcaiger) [inline packages/web-frontend/src/actions.js:907]: remove redux state and handle open state in component

File: packages/web-frontend/src/components/mobile/Dashboard/index.js [hunk 2]
```diff
@@ -5,130 +5,86 @@
...
+
+export const Dashboard = ({
+  dashboards,
+  orgUnit,
+  toggleFilter,
+  filterIsExpanded,
+  currentDashboardName,
+  handleFilterChange,
+}) => {
+  const [isDashboardExpanded, setIsDashboardExpanded] = useState(true);
```
Comment (tcaiger) [inline packages/web-frontend/src/components/mobile/Dashboard/index.js:30]: Set open as default

File: packages/web-frontend/src/containers/OverlayDiv/components/ProjectPage/index.js [hunk 3]
```diff
@@ -23,10 +23,14 @@ import { ProjectCard } from './ProjectCard';
...
 
 const Container = styled.div`
-  margin: 24px 0;
   display: grid;
   grid-template-columns: repeat(3, 1fr);
   gap: 16px;
+  margin: 24px 0;
+
+  @media (max-width: 600px) {
+    grid-template-columns: 1fr;
```
Comment (tcaiger) [inline packages/web-frontend/src/containers/OverlayDiv/components/ProjectPage/index.js:32]: Responsive style for project selector

File: packages/web-frontend/src/containers/mobile/SearchOverlay/index.js [hunk 4]
```diff
@@ -116,14 +116,14 @@ SearchOverlay.propTypes = {
 };
 
 const mapStateToProps = state => {
-  const { isLoading, searchString, searchResponse } = state.searchBar;
+  const { isLoadingSearchResults, searchString, searchResults } = state.searchBar;
```
Comment (tcaiger) [inline packages/web-frontend/src/containers/mobile/SearchOverlay/index.js:119]: fix search results mapping

File: packages/web-frontend/src/containers/OverlayDiv/components/ProjectPage/index.js (hunk 3 above)
Comment (edmofro) [reply packages/web-frontend/src/containers/OverlayDiv/components/ProjectPage/index.js:32]: Beautiful, so simple!

**Review by edmofro — APPROVED** [review]
All looks awesome, thanks for getting this done before your break!

**Pull request merged**